src/Microsoft.AspNetCore.Mvc.Razor/Internal/DefaultRazorProject.cs

```diff
@@ -41,6 +42,8 @@ public override IEnumerable<RazorProjectItem> EnumerateItems(string path)
             return EnumerateFiles(_provider.GetDirectoryContents(path), path, "");
         }
 
+        public virtual IChangeToken Watch(string pattern) => _provider.Watch(pattern);
```

Comment:
We'll figure it out 👍 I just wanted to prevent heartbreak. **NO additional dependencies for Razor**